### PR TITLE
chore: adding new version of Argo cert chart

### PR DESCRIPTION
**Pull request by badal773**
Here we have added an option for if the cronjob will run after specify date you will get alert and added the new script for cert refresh

### CODE REVIEW

**Review by ajaydevtron — CHANGES_REQUESTED** [review]

File: charts/argocd-certificate-refresh/values.yaml [hunk 1]
```diff
@@ -4,23 +4,29 @@
...
 
 
 
 #set up this If you need alert for argocd cert expire
 alert:
   enabled: false
-  image: quay.io/devtron/k8s-utils:3f8c7b04-489-16490
-  discord_url: " " 
-  schedule: "59 23 * */6 *"
+  image: quay.io/devtron/test:578de80c-489-21336
```
[the remark this answers is not in the document]
Comment (badal773) [reply charts/argocd-certificate-refresh/values.yaml]: done

File: charts/argocd-certificate-refresh/values.yaml [hunk 2]
```diff
@@ -4,23 +4,29 @@
 
 
 # docker image information
-image: quay.io/devtron/argocd-cert-refresh:v7
+image: quay.io/devtron/test:a2a56bd1-555-21342
```
[the remark this answers is not in the document]
Comment (badal773) [reply charts/argocd-certificate-refresh/values.yaml]: done

**Review by ajaydevtron — APPROVED** [review]

File: charts/argocd-certificate-refresh/templates/role.yaml [hunk 3]
```diff
@@ -2,14 +2,21 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   name: argocd-cert-role
+  namespace: devtroncd
```
Comment (prakarsh-dt) [inline charts/argocd-certificate-refresh/templates/role.yaml:5]: Make this configurable from values.yaml

Comment (badal773) [reply charts/argocd-certificate-refresh/templates/role.yaml:5]: In the script, we are deleting secrets and pods of argocd from devtroncd namespace `argocd-secret -n devtroncd`
Does it need to be configured or all the configuration is only valid for Devtron pod
https://github.com/devtron-labs/devtron-utilities/pull/57

File: charts/argocd-certificate-refresh/templates/argo-cert-expire-alert-job.yaml [hunk 4]
```diff
@@ -7,8 +7,10 @@ apiVersion: batch/v1beta1
 kind: CronJob
 metadata:
   name: argocd-cert-alert
+  namespace: {{ $.Values.argocd_namespace| default .Release.Namespace }}
```
Comment (ajaydevtron) [inline charts/argocd-certificate-refresh/templates/argo-cert-expire-alert-job.yaml:10]: use this variable argocdNamespace

File: charts/argocd-certificate-refresh/templates/argo-cert-expire-alert-job.yaml [hunk 5]
```diff
@@ -23,12 +25,17 @@ spec:
           - name: argocd-alert-container
             image:  {{ $.Values.alert.image }}
             imagePullPolicy: IfNotPresent
-            command: ["/bin/sh", "-c", "chmod +x alert.sh && ./alert.sh"]
             env:
+              - name: devtron_namespace
+                value: {{ $.Values.devtron_namespace| default .Release.Namespace }}
```
Comment (ajaydevtron) [inline charts/argocd-certificate-refresh/templates/argo-cert-expire-alert-job.yaml:30]: Use the devtronNamespace variable instead of devtron_namespace

**Review by ajaydevtron — APPROVED** [review]

**Review by ajaydevtron — APPROVED** [review]

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**